pulser-core/pulser/devices/_device_datacls.py

```diff
@@ -595,67 +595,70 @@ def _specs(self, for_docs: bool = False) -> str:
         def yes_no_fn(cond: bool) -> str:
             return "Yes" if cond else "No"
 
-        lines = [
-            "\nRegister parameters:",
-            f" - Dimensions: {self.dimensions}D",
-            f" - Rydberg level: {self.rydberg_level}",
-            f" - Maximum number of atoms: {self.max_atom_num}",
-            f" - Maximum distance from origin: {self.max_radial_distance} μm",
-            (
-                " - Minimum distance between neighbouring atoms: "
-                f"{self.min_atom_distance} μm"
-            ),
-            f" - Maximum layout filling fraction: {self.max_layout_filling}",
-            f" - SLM Mask: {yes_no_fn(self.supports_slm_mask)}",
-        ]
-
-        if self.max_sequence_duration is not None:
-            lines.append(
-                " - Maximum sequence duration: "
-                f"{self.max_sequence_duration} ns"
+        def check_none(var: Any, line: str) -> str:
+            if var is None:
+                return ""
+            else:
+                return line.format(var)
+
+        reg_lines = (
+            "\nRegister parameters:\n"
+            + f" - Dimensions: {self.dimensions}D\n"
+            + f" - Rydberg level: {self.rydberg_level}\n"
+            + check_none(self.max_atom_num, " - Maximum number of atoms: {}\n")
+            + check_none(
+                self.max_radial_distance,
+                " - Maximum distance from origin: {} µm\n",
```

Comment:
If you prefer, I can change back, but I can give some more arguments. What I did not like with list, is that there were three different syntaxes – `lines = [...]`, `lines.append()`, `lines += []` – and one had to go back and forth between each, which is quite verbose. With a string, there is a single syntax, except when I need to check if an attribute is present (but this is caused by layout, and this would be unnecessary if I implemented your suggestion of having `layout_lines` only for `Device`).

Reply:
These are good arguments, I will revert back to lists.